client/aliases.go

```diff
@@ -81,6 +81,8 @@ func (client *Client) Unalias(aliasOrSnap string) (changeID string, err error) {
 type AliasStatus struct {
 	App    string `json:"app,omitempty"`
 	Status string `json:"status,omitempty"`
```

Comment:
So we discussed on IRC and the conclusion is to keep status, but consider later exposing the real internal logic bits over /snap

also goign to s/App/Command/, s/app/command/ in the struct here